specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -138,6 +140,16 @@ supports customers that want to:
 1. Use the shared component libraries in their projects.
 2. Build their own docker image for data-pipelines
 
+#### Docker
+A Docker image will be published on docker hub under the user https://hub.docker.com/u/versatiledatakit.
+It is built by the CI on any changes to the frontend codebase.
```

Comment:
How does the frontend application image requirements differ from the frontend test image already produced?

Reply:
The frontend test image includes browsers and testing utils. 
This is completely needless bloat for the production system. 
In the production image we just include the distribution folder from the build directory and nginx server.

Reply:
May add the requirements known (about the image) to the tickets for creating the runtime image.